src/main/java/rx/internal/util/ScalarSynchronousObservable.java

```diff
@@ -33,6 +34,8 @@
  * @param <T> the value type
  */
 public final class ScalarSynchronousObservable<T> extends Observable<T> {
+    /** The execution hook instance. */
+    static RxJavaObservableExecutionHook hook = RxJavaPlugins.getInstance().getObservableExecutionHook();
```

Comment:
Ah, you override it in the test. Then it needs a comment or test should use reflection.